tests/apollo/util/bft.py

```diff
@@ -1766,7 +1766,7 @@ async def expected_view_to_be_reached():
                 key = ['replica', 'Gauges', 'view']
                 replica_view = await self.retrieve_metric(replica_id, *key)
 
-                if replica_view is not None and replica_view == expected_view:
+                if replica_view is not None and replica_view >= expected_view:
```

Comment:
I don't think this statement has to be modified as it aims to prove that the given replica has reached exactly the specified view. The function `async def wait_for_replica_to_reach_view` was created as a helper of the function `async def wait_for_replicas_to_reach_view`, which opens a nursery where all of the replicas are tested to be in the same view. Changing this line would break/invalidate the guarantees that we were relying on in the tests that were using the function `async def wait_for_replicas_to_reach_view` before.

My suggestion is to create a separate function, which checks whether _the expected view or a higher one_ is reached or not.

Reply:
I agree with Stefan, we have many test cases in CI that for example arrange the situation to only do a single View Change with 1 View increment. Might be it will be best to create another function that loosens up the expectations for the longer running tests.